modules/store/spec/selector.spec.ts

```diff
@@ -123,6 +123,106 @@ describe('Selectors', () => {
     });
   });
 
+  describe('createSelector with props', () => {
```

Comment:
We need some integration tests using `store.select` also.

Reply:
Thanks for reminding me!
Creating this test made me realize we also have to change the `select` function.
Doing this I bumped into a small problem, see my comment in the following commit.

Reply:
I think only having props for selector functions is reasonable.